lib/discovery.ts

```diff
@@ -0,0 +1,44 @@
+
+import { SerialPort } from "serialport";
+import {RevHub} from "./RevHub.js";
+import {openRevHub} from "./internal/RevHub.js";
+
+export async function getSerialPortPathForExHubSerial(
```

Comment:
This should not be exported in `index.ts`.

Reply:
Moving this inside of the open-rev-hub file, and removing the export.